doc/api/cli.md

```diff
@@ -121,6 +121,13 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--redirect-warnings=file`
```

Comment:
I'm not terribly convinced this is useful enough to justify.

1. It can't be triggered by env var, so its hard to configure in the cloud, or as a local user policy for people who just don't care and don't want to see the warnings
2. warnings are emitted as events on process, specifically so this kindof handling can be implemented by the user, so why are we doing it for them?
3. why just warnings? what about an option to redirect stack traces on uncaught errors or exceptions?
4. why not just redirect stderr to a file?